nav2_navfn_planner/include/nav2_navfn_planner/navfn_planner.hpp

```diff
@@ -111,8 +111,11 @@ class NavfnPlanner : public nav2_util::LifecycleNode
 
   // Request costmap from world model
   void getCostmap(
-    nav2_msgs::msg::Costmap & costmap, const std::string layer = "master",
-    const std::chrono::milliseconds waitTime = std::chrono::milliseconds(100));
+    nav2_msgs::msg::Costmap & costmap,
+    const std::string layer = "master");
+
+  // get latest robot pose from the world model
+  geometry_msgs::msg::Pose getRobotPose();
```

Comment:
why is `getRobotPose` being called here in the header like this?

Reply:
It's a function prototype here, not a call.